index.html

```diff
@@ -1277,9 +1277,14 @@ <h2>Security Considerations</h2>
       </ul>
     </section>
 
-    <section class='appendix'>
+    <section id="acknowledgments" class="informative appendix">
       <h2>Acknowledgments</h2>
       <p>
+        The following people have been instrumental in providing thoughts, feedback, reviews, content, criticism and
+        input in the creation of this specification:
+      </p>
+      <p style="margin-left: 3em;">
+        Sarven Capadisli, Herbert Van de Sompel, the Fedora Repository, Islandora and Samvera communities.
```

Comment:
The communities aren't people, and there is an ambiguity in the list as to which elements "communities" applies to. Perhaps:

> Sarven Capadisli, Herbert Van de Sompel, **and members of** the Fedora Repository, Islandora and Samvera communities.